src/Resources/config/services.yml

```diff
@@ -92,6 +92,17 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.store.session:
```

Comment:
Changing it would greatly help to implement advanced security (see #122). I think we should change both if we change one of them…